static/styles/recent_topics.css

```diff
@@ -262,6 +287,14 @@
 
         .recent_topic_name {
             width: 40%;
+
+            .line_clamp {
+                /* stylelint-disable-next-line value-no-vendor-prefix */
```

Comment:
We should include an explanation of why we think the rule should not apply here when doing these overrides.